README.md

```diff
@@ -1,11 +1,11 @@
 # Manim-Tutorial
 A tutorial for manim, a mathematical animation engine made by 3b1b for Python. 
-## Requirements
```

Comment:
You don't have to entirely remove the req section. This guide is for everyone, including beginners. In the past, there have been beginners that have used this. So, leave the req section with Python as one of the reqs, and change linux to mac, linux or window.

Reply:
Keep Original. I will change it